docs/faqs.md

```diff
@@ -0,0 +1,23 @@
+# FAQs #
+
+##Cloning questions##
```

Comment:
The only concern with putting satellite-clone here is that is the name of the repo  so it may get confusing. We could do `satellite clone playbook` I suppose, but my thought around this FAQ is to put everything in layman's terms